astropy/modeling/functional_models.py

```diff
@@ -786,6 +786,53 @@ def inverse(self):
         return self.__class__(slope=new_slope, intercept=new_intercept)
 
 
+class Planar2D(Fittable2DModel):
```

Comment:
If so, why isn't the same done with Linear1D? It's in the same spirit as that one.